tests/Feature/Proxy/CarrierProxyTest.php

```diff
@@ -2,7 +2,7 @@
 
 namespace MyParcelCom\ApiSdk\Tests\Feature\Proxy;
 
-use GuzzleHttp\ClientInterface;
+use MyParcelCom\ApiSdk\Http\Contracts\HttpClient\ClientInterface;
```

Comment:
Should this not be `Http\Client\HttpClient`?

It seems like it needs to be changed in a lot of tests.

Reply:
That is interesting. I removed the interface with the "safe" method. How are these tests not failing with the interface no longer existing? :confused:

Reply:
I think it's because the interface is only used in a docblock.